press-kit/index.md

```diff
@@ -120,7 +120,7 @@ meta_descr: meta_descr.presskit
   <div class="info-block">
     <h2>{% t press-kit.pressdoc %}</h2>
     <div>
-        <h3><a href="https://www.monerooutreach.org/quick-facts.html">{% t press-kit.quickfacts %}</a></h3>
+        <h3><a href="Monero_QuickFacts_June_2024.pdf">{% t press-kit.quickfacts %}</a></h3>

```

Comment:
```suggestion
        <h3><a href="Monero_QuickFacts_June_2024.pdf">{% t press-kit.quickfacts %} (PDF)</a></h3>
```


does this work? - added ` (PDF)` so people know that its a file. I assume PDF doesn't need to be translated.

otherwise, LGTM

Reply:
Good point. Presenting direct link to DL/open a .pdf (especially to our audience) is, at the very least, not very courteous: see https://stackoverflow.com/a/48431884

Reply:
pointing to https://resilience365.com/monero-quick-facts/ now. the pdf remains for archival purposes only.